### PR TITLE
Update slurm_cancel.sh

### PR DESCRIPTION
**Pull request by efajardo**
There was a bug caught in HCodnor Ticket [5804](https://htcondor-wiki.cs.wisc.edu/index.cgi/tktview?tn=5804) . This fixes it for [SOFTWARE-TICKET-2404](https://jira.opensciencegrid.org/browse/SOFTWARE-2404).

However this has been shown hard to test for the reasons mentioend [here](https://jira.opensciencegrid.org/browse/SOFTWARE-2404?focusedCommentId=338319&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-338319)

Basically it does not affect any of our current users. But it is good to have it fixed.  Can you please merge this @djw8605 and let me know if you got any concerns.

### CODE REVIEW

**Comment by djw8605** [comment]
Actually, I'm the one that reported this bug to the HTCondor team.  I'm glad it is fixed.  Looks good to me!

**Comment by efajardo** [comment]
Awesome @djw8605 can you also tag it?

**Comment by djw8605** [comment]
tagged, [v1.18.23.bosco](https://github.com/osg-bosco/BLAH/releases/tag/v1.18.23.bosco)
